### PR TITLE
enh(mc): automatic commit - Open LDAP

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

REFS: CTOR-405

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-openldap-ldap.md [hunk 1]
```diff
@@ -2,37 +2,334 @@
...
+| system.operations.delete.completed.count | count |
 
-### Remote server
+</TabItem>
+</Tabs>
+
+## Prerequisites
 
 The remote server must have an LDAP service running and available. A user and
 password dedicated to monitoring is needed for security reasons.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-openldap-ldap.md:55]:
```suggestion
password dedicated to monitoring are needed for security reasons.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-openldap-ldap.md [hunk 2]
```diff
@@ -2,37 +2,335 @@
...
-```
+Le connecteur apporte le modèle de service suivant
+(classé selon le modèle d'hôte auquel il est rattaché) :
+
+<Tabs groupId="sync">
+<TabItem value="App-Openldap-LDAP-custom" label="App-Openldap-LDAP-custom">
+
+| Alias        | Modèle de service                     | Description                                                   |
+|:-------------|:--------------------------------------|:--------------------------------------------------------------|
+| System-Usage | App-Openldap-System-Usage-LDAP-custom | Contrôler l'utilisation du Système OpenLDAP via LDAP Protocol |
```
[the remark this answers is not in the document]
Comment (lucie-dubrunfaut) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-openldap-ldap.md:24]: Je crois ce [LDAP](https://fr.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol) là daprès la page wiki [OpenLDAP](https://fr.wikipedia.org/wiki/OpenLDAP)

Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-openldap-ldap.md:24]:
```suggestion
| System-Usage | App-Openldap-System-Usage-LDAP-custom | Contrôler l'utilisation du système OpenLDAP via le protocole LDAP |
```

File: pp/integrations/plugin-packs/procedures/applications-openldap-ldap.md [hunk 3]
```diff
@@ -2,37 +2,334 @@
...
-yum install centreon-plugin-Applications-OpenLDAP-Ldap
-```
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Openldap-LDAP-custom" label="App-Openldap-LDAP-custom">
+
+| Service Alias | Service Template                      | Service Description                       |
+|:--------------|:--------------------------------------|:------------------------------------------|
+| System-Usage  | App-Openldap-System-Usage-LDAP-custom | Check System Usage of OpenLDAP using LDAP |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-openldap-ldap.md]:
```suggestion
| System-Usage  | App-Openldap-System-Usage-LDAP-custom | Monitor system usage of OpenLDAP using the LDAP protocol |
```

File: pp/integrations/plugin-packs/procedures/applications-openldap-ldap.md [hunk 4]
```diff
@@ -2,37 +2,334 @@
...
+* **App-Openldap-LDAP-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="App-Openldap-LDAP-custom" label="App-Openldap-LDAP-custom">
+
+| Service Alias | Service Template                      | Service Description                       |
+|:--------------|:--------------------------------------|:------------------------------------------|
+| System-Usage  | App-Openldap-System-Usage-LDAP-custom | Check system usage of OpenLDAP using LDAP |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-openldap-ldap.md:23]:
```suggestion
| System-Usage  | App-Openldap-System-Usage-LDAP-custom | Check system usage of OpenLDAP using the LDAP protocol |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**